### DISCUSSION TITLE
Stockfish Material counting

### DISCUSSION THREAD

**Discussion by fishchess** [official-stockfish/Stockfish#4251] [Q&A] [answered]
I see material value big , for example Knight value is 781 but i know Knight value is 325

```
  PawnValueMg   = 126  
  KnightValueMg = 781 
  BishopValueMg = 825   
  RookValueMg   = 1276
  QueenValueMg  = 2538
```

How to convert this value to real value? 

Are they divided into numbers somewhere?

**Comment by Disservin** [comment]
These are stockfishs internal values.
It adjusts the score when it reports back the info
score * 100 / 208 (old but still relative for HCE)

note: there’s no fixed value a piece should have

**Comment by vondele** [comment, marked as answer]
In the classical evaluation, the convention was to divide by PawnValueEg, which was 208, to convert to pawns.
